web/cobrands/sass/_mixins.scss

```diff
@@ -1,5 +1,17 @@
 $direction: 'left' !default;
 
+// Padding
+$pt: .75em;
```

Comment:
These 8 variables have very generic names for something in the global scope!! I’d err towards longer, more obvious variable names, to aid with understanding/debugging later on, rather than super short names. Something like `$button-margin-top` etc.

If these are defaults, I’d also give them a `!default` flag, to make them easier for cobrands to override.